src/components/ContactsList/_modals/VerifyModal/PeerTOTP.tsx

```diff
@@ -0,0 +1,53 @@
+import { confirmContactWithTOTP } from '@/api'
+import { ErrorBar } from '@/components/ErrorBar'
+import { Input } from '@/components/Input'
+import { FC, useMemo, useState } from 'react'
+
+const TOTP_VALID_REGEX = /^\d{2}$/
+
+export const usePeerTOTP = (peerPaymail: string, onConfirmed: () => void) => {
+  const [value, setValue] = useState('')
+  const [confirming, setConfirming] = useState(false)
```

Comment:
Please use isLoading, setIsLoading name

Reply:
OK 👍 But to:
`const [loading, setLoading] = useState(false)`

because as I can see - such is the convention in this repo